### ISSUE TITLE
Refactor

### ISSUE THREAD

**Issue by Sternenwarte88** [Org-Helping-Hands/react-app#27]
An Issue to refactor to functional approach and remove duplicate imports. Please assign it to me :)

**Comment by thisisommore** [comment]
:)

**Comment by Sternenwarte88** [comment]
Thanks :)

**Comment by Sternenwarte88** [comment]
So for this moment, I set up eslint and prettier. Eslint with some standart rules for typescript, combined with prettier it should be easy, to be consense and get clear rules for developing :)

**Comment by thisisommore** [comment]
Great, eslint with vscode or any additional configs ?

**Comment by thisisommore** [comment]
Oh standard rules, I did configured in Angular, eslint --fix was working there. Hope that works here too now. Before I tired it didn't fixed warnings.

**Comment by thisisommore** [comment]
I am addressing some lint warnings here #24 😉

**Comment by Sternenwarte88** [comment]
Ay saw this now. Ehm got some Errors from TS compiler -.- 
I'm check out those lint warnings. We have to change a bit of code. Most things are unused. But there are some, like don´t use new because of side effects or don´t throw errors x.x 

Have to look closer on it x.x So i suggest don´t merge it for now.

**Comment by Sternenwarte88** [comment]
Oh sorry i forgot. I use eslint, prettier and eslint prettier extensions from vscode. [Eslint prettier](https://marketplace.visualstudio.com/items?itemName=rvest.vs-code-prettier-eslint) is a really good extension. This does first formatting and then run eslint actions on it for those known formatter/eslint problems.

**Comment by thisisommore** [comment]
Yeah, it will be in draft for long time 😂, make sure to not do any changes I did there, or continue from there

**Comment by thisisommore** [comment]
Great, will try that extension then, does it fixes warnings of eslint rules ?

**Comment by Sternenwarte88** [comment]
I try xD
Yeah it would solve some oh those problems, sadly not all.
